config/check.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+set -e
+
+CLUSTERS="mainnet staging"
```

Comment:
This duplicates what clusters are configured at multiple places, creating no benefit for having a separate CI check if dhall is evaluated as part of CI anyway.

Reply:
@domenkozar, this is for quick local checks, during development

Reply:
It shouldn't introduce duplicated list again - we've made this mistake too many times.